plugin-server/src/cdp/legacy-plugins/_destinations/customerio/index.ts

```diff
@@ -108,16 +107,8 @@ export const setupPlugin = async (meta: CustomerIoMeta) => {
         EVENTS_CONFIG_MAP[config.sendEventsFromAnonymousUsers || DEFAULT_SEND_EVENTS_FROM_ANONYMOUS_USERS]
     global.identifyByEmail = config.identifyByEmail === 'Yes'
 
-    const credentialsVerifiedPreviously = await storage.get(global.authorizationHeader, false)
```

Comment:
This was actually bad as it was a permanent storage thing that led to a now invalid integration not being marked as such